src/50_Sample_AMPs/Product_Sample.html

```diff
@@ -15,266 +15,286 @@
+            "ratingValue": "4.4",
+            "reviewCount": "88"
+        },
+        "offers": {
+            "@type": "Offer",
+            "priceCurrency": "USD",
+            "price": "500",
+            "priceValidUntil": "2020-11-05",
+            "itemCondition": "http://schema.org/UsedCondition",
+            "availability": "http://schema.org/InStock",
+            "seller": {
+                "@type": "AMPByExample",
+                "name": "AMPByExample"
+            }
+        }
+    }
+    </script>
 
-    <!-- This button adds the item to the shopping cart. Pressing this button should lead to a the shopping cart page on your normal website and the cart should be prefilled with the current item. -->
-    <p id="product-header">
-        <h2>Nexus 6P</h2>
-        <a href="/shopping-cart-sample.html" class="button button-primary call-to-action">Buy</a>
-    </p>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    </head>
+    <body>
+        <!--
+        Use `amp-img` to add the logo of the seller.
+        -->
+        <div class="seller-logo">
+            <amp-img src="/img/google_store.png" width="163" height="35"></amp-img>
+        </div>
 
-    <p id="product-description">
-        A 5.7-inch, high-resolution WQHD AMOLED display and front-facing stereo speakers — all housed in a contoured aluminum body.
-    </p>
+        <!-- This button adds the item to the shopping cart. Pressing this button should lead to a the shopping cart page on your normal website and the cart should be prefilled with the current item. -->
+        <p id="product-header">
+            <h2>Nexus 6P</h2>
+            <a href="/shopping-cart-sample.html" class="button button-primary call-to-action seller-button">Buy</a>
+        </p>
 
-    <!-- #### Image Gallery -->
-    <!-- The `amp-carousel` works very well for product image galleries. Learn more about `amp-carousel` [here](/advanced/image_galleries_with_amp-carousel/). -->
+        <p id="product-description">
+            A 5.7-inch, high-resolution WQHD AMOLED display and front-facing stereo speakers — all housed in a contoured aluminum body.
+        </p>
+        <h5 class="price-description">From 399.00$</h5>
```

Comment:
Prices are often dynamic which can be a problem if the page is accessed via the AMP Cache. Could we use amp-list to include a dynamic price?